tests/safeds_stubgen/stubs_generator/test_generate_stubs.py

```diff
@@ -100,6 +100,10 @@ def test_abstract_creation(snapshot_sds_stub: SnapshotAssertion) -> None:
     assert_stubs_snapshot("abstract_module", snapshot_sds_stub)
 
 
+def test_inheritance_creation(snapshot_sds_stub: SnapshotAssertion) -> None:
+    assert_stubs_snapshot("inheritance_module", snapshot_sds_stub)
+
+
```

Comment:
Having to add another test case for each test module is not ideal. Better discover all `py` files in `tests/data` that you want to create stubs for using [`glob`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.glob) and use one parametrized test.

Reply:
Wouldn't the snapshot tests only create one test file, if we'd use just one parametrized test, since it creates one file for each test that is called?

Reply:
We're using parametrized snapshot tests in the library, e.g. to test [methods of `Image`](https://github.com/Safe-DS/Library/blob/main/tests/safeds/data/image/containers/test_image.py). It works fine.

Reply:
Resolving this in PR #70